presto-mongodb/src/test/java/io/prestosql/plugin/mongodb/TestMongoDistributedQueries.java

```diff
@@ -53,20 +53,6 @@ protected boolean supportsViews()
         return false;
     }
 
-    @Override
-    public void testCreateTable()
-    {
-        // TODO https://github.com/prestosql/presto/issues/3082
-        throw new SkipException("Fix DROP TABLE");
-    }
-
-    @Override
-    public void testCreateTableAsSelect()
-    {
-        // TODO https://github.com/prestosql/presto/issues/3082
-        throw new SkipException("Fix DROP TABLE");
```

Comment:
For the future, i suggest we disable tests in more "auto-managed" way, like

```
assertThatThrownBy(() -> super.test())
  .hasMessage("foo bar");
```

that way we won't forget to uncomment them

Reply:
@findepi Agreed, it's a good idea. Let me filter with it from next time.